docker-bake.hcl

```diff
@@ -28,6 +28,15 @@ target "test-debian" {
     }
 }
 
+target "test-rhel" {
+    inherits = ["test-base"]
+    args = {
+        TEST_BASE_TYPE = "rhel"
+        TEST_BASE_IMAGE = "fedora:35"
+        TEST_CMDS = "info"
```

Comment:
Remove this line in a follow-up when windres test is fixed for this case:

```
#24 [test  6/11] RUN --mount=type=cache,target=/pkg-cache     [ "${TEST_CMDS:-windres}" = "${TEST_CMDS#*windres}" ] && exit 0; ./test-windres.bats
#24 0.173 1..3
#24 0.254 not ok 1 invalid
#24 0.254 # (from function `xxrun' in file test_helper.bash, line 54,
#24 0.254 #  from function `add' in file test_helper.bash, line 7,
#24 0.254 #  in test file test-windres.bats, line 7)
#24 0.255 #   `add llvm' failed with status 127
#24 0.255 # /work/test_helper.bash: line 54: apt: command not found
#24 0.319 not ok 2 basic
#24 0.319 # (from function `assert_success' in file bats-assert/src/assert.bash, line 114,
#24 0.319 #  in test file test-windres.bats, line 20)
#24 0.319 #   `assert_success' failed
#24 0.319 #
#24 0.319 # -- command failed --
#24 0.319 # status : 1
#24 0.319 # output : llvm-rc not installed
#24 0.319 # --
#24 0.320 #
#24 0.359 ok 3 clean
#24 ERROR: process "/bin/sh -c [ \"${TEST_CMDS:-windres}\" = \"${TEST_CMDS#*windres}\" ] && exit 0; ./test-windres.bats" did not complete successfully: exit code: 1
------
 > [test  6/11] RUN --mount=type=cache,target=/pkg-cache     [ "${TEST_CMDS:-windres}" = "${TEST_CMDS#*windres}" ] && exit 0; ./test-windres.bats:
#24 0.319 # (from function `assert_success' in file bats-assert/src/assert.bash, line 114,
#24 0.319 #  in test file test-windres.bats, line 20)
#24 0.319 #   `assert_success' failed
#24 0.319 #
#24 0.319 # -- command failed --
#24 0.319 # status : 1
#24 0.319 # output : llvm-rc not installed
#24 0.319 # --
#24 0.320 #
#24 0.359 ok 3 clean
```

Reply:
I added a check for this in https://github.com/tonistiigi/xx/pull/49/files#diff-97b6828de99e6a75768dba426569ce1a89f0e32542b0ea9c7a3349f47a99263bR16 . Does centos really ship with very old llvm or why isn't this installed?

Reply:
> Does centos really ship with very old llvm or why isn't this installed?

I have not yet updated the [test_helper.bash](https://github.com/tonistiigi/xx/blob/master/base/test_helper.bash) to include yum/dnf. I can add the add/del implementation so the test would pass but we might also need to add `xx-yum` or `xx-dnf`.

Reply:
Also centos and rhel don't have the `bats` pkg available in their main repo so I might need to fix sources like you in #49